es/src/main/scala/monix/connect/es/Elasticsearch.scala

```diff
@@ -1,141 +1,211 @@
 package monix.connect.es
 
+import cats.effect.Resource
+import com.sksamuel.elastic4s.http.JavaClient
 import com.sksamuel.elastic4s.requests.bulk.{BulkCompatibleRequest, BulkResponse}
 import com.sksamuel.elastic4s.requests.count.{CountRequest, CountResponse}
 import com.sksamuel.elastic4s.requests.delete._
+import com.sksamuel.elastic4s.requests.get.{GetRequest, GetResponse}
 import com.sksamuel.elastic4s.requests.indexes._
 import com.sksamuel.elastic4s.requests.indexes.admin.{DeleteIndexResponse, RefreshIndexResponse}
-import com.sksamuel.elastic4s.requests.searches.{SearchRequest, SearchResponse}
+import com.sksamuel.elastic4s.requests.searches.{SearchHit, SearchRequest, SearchResponse}
 import com.sksamuel.elastic4s.requests.update.{UpdateRequest, UpdateResponse}
-import com.sksamuel.elastic4s.{ElasticClient, Response}
+import com.sksamuel.elastic4s.{ElasticClient, ElasticProperties, HttpClient, Response}
 import monix.eval.Task
+import monix.execution.annotations.UnsafeBecauseImpure
+import monix.reactive.{Consumer, Observable}
 
 /**
-  * Object for managing the Elasticsearch
+  * Singleton object provides builders for [[Elasticsearch]].
+  *
+  * ===Example===
+  *{{{
+  * import com.sksamuel.elastic4s.ElasticDsl._
+  * import com.sksamuel.elastic4s._
+  *
+  * val esResource = Elasticsearch.create("http://localhost:9200")
```

Comment:
very nice to see examples like this one! :D

Reply:
Coming soon